cypress/e2e/shared/explorer.queries.graphs.test.ts

```diff
@@ -169,13 +176,6 @@ describe.skip('writing queries and making graphs using Data Explorer', () => {
     })
 
     it('shows the empty state when the query returns no results', () => {
-      cy.isIoxOrg().then(isIox => {
-        // iox uses `${orgId}_${bucketId}` for a namespace_id
-        // And gives a namespace_id failure if no data is written yet.
-        // https://github.com/influxdata/monitor-ci/issues/402#issuecomment-1362368473
-        cy.skipOn(isIox)
-      })
-
```

Comment:
No need to skip anymore since the bug is fixed by https://github.com/influxdata/idpe/pull/16911 that integrate IOx namespace into Cloud bucket API

Reply:
sweet!